pkg/ccl/backupccl/backup_job.go

```diff
@@ -350,7 +350,8 @@ func backup(
 	var tableStatistics []*stats.TableStatisticProto
 	for i := range backupManifest.Descriptors {
 		if tableDesc := sqlbase.TableFromDescriptor(&backupManifest.Descriptors[i], hlc.Timestamp{}); tableDesc != nil {
-			// Collect all the table stats for this table.
+			// Invalidate the stats for this table to ensure they are up to date.
+			statsCache.InvalidateTableStats(ctx, tableDesc.GetID())
```

Comment:
Hm. 

Could this also mean other traffic sees a blip every time backup runs and evicts the cache?
Also, I guess compared to all the actual data exporting backup does blocking on re-reading stats shouldn't be _that_ bad, but do we have a sense for how bad it might be for e.g. 1k tables doing small/fast inc backups?

Reply:
Ya, I also felt a bit uneasy. Although I don't think the perf hit would be too bad, it's a bit of an uneasy backport. I found that directly inject statistics invalidates the stats cache so instead I opted to just load stats like that to unflake the tests.